submarine-server/server-core/src/test/java/org/apache/submarine/jobserver/rest/api/JobApiTest.java

```diff
@@ -17,111 +17,104 @@
  * under the License.
  */
 
-import com.google.gson.Gson;
-import org.apache.submarine.jobserver.rest.dao.JsonResponse;
-import org.apache.submarine.jobserver.rest.dao.RestConstants;
-import org.apache.submarine.jobserver.rest.api.JobApi;
-import org.glassfish.jersey.server.ResourceConfig;
-import org.glassfish.jersey.test.JerseyTest;
-import org.junit.Test;
-
-import javax.ws.rs.client.Entity;
-import javax.ws.rs.core.Application;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
-
-import static org.junit.Assert.assertEquals;
+package org.apache.submarine.jobserver.rest.api;
 
-public class JobApiTest extends JerseyTest {
+import org.junit.Test;
 
+public class JobApiTest { //extends JerseyTest {
+  /*
```

Comment:
Why do we need to comment on this test class? The test cases are not running after this modification

Reply:
The test cases here have no practical meaning.
I created new JIRA [SUBMARINE-293. Add submarine server rest test case](https://issues.apache.org/jira/browse/SUBMARINE-293)